bundles/org.openhab.voice.porcupineks/src/main/java/org/openhab/voice/porcupineks/internal/PorcupineKSConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.voice.porcupineks.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link PorcupineKSConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Miguel Álvarez - Initial contribution
+ */
+@NonNullByDefault
+public class PorcupineKSConfiguration {
+
+    /**
+     * Sample configuration parameters. Replace with your own.
+     */
+    public String apiKey = "";
+    /**
+     * Sample configuration parameters. Replace with your own.
+     */
+    public float sensitivity = 0.5f;
```

Comment:
I am not sure the javadoc for each of your constants has any value here. Either you suppress it, or you provide a clearer description.

Reply:
Sorry I forgot to write those, ty for pointing at it.